test/fixtures/invalid/init_crash.yml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: init-crash
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        name: web
+        app: crash-app
+    spec:
+      initContainers:
+      - name: init-crash-loop-back-off
```

Comment:
That's a good point; there isn't currently any test with a successful init container.